components/openapi/src/main/scala/pl/touk/nussknacker/openapi/parser/ParseToSwaggerService.scala

```diff
@@ -138,7 +133,15 @@ private[parser] class ParseToSwaggerService(openapi: OpenAPI, openAPIsConfig: Op
       .map(_._2)
   }
 
-  private def requestParameter(request: RequestBody): ValidationResult[Option[SwaggerParameter]] = {
+  private def findContentType(content: Content): Option[String] = {
+    val scalaVersion = content.asScala
+    scalaVersion
+      .find(_._1.startsWith("application/json"))
+      .orElse(scalaVersion.find(_._1.contains("*/*")))
```

Comment:
can we extract common part from this method and `findMediaType`?

Reply:
done